docs/configuration.md

```diff
@@ -20,6 +20,23 @@ The following table shows the available config keys for Deptrac.
 </thead>
 <tbody>
 <tr>
+<td>analyser.internal_tag</td>
+<td>
+Specifies which doc block tag deptrac should use to identify layer-internal class-like tokens.
+The default is <code>"@internal"</code>. May be set to <code>null</code> to disable.
+Note that the tag <code>@deptrac-internal</code> will always be used to identify
+layer-internal class-like tokens, even if this <code>internal_tag</code> is set to <code>null</code>.
+</td>
+<td>
+
+```yaml
+deptrac:
+  analyser:
+    internal_tag: ~
```

Comment:
This needs to be also possible via the `php-config`. Because the `yaml` config will be deprecated soon.

Reply:
Ah, interesting!

What do I need to do to make it possible, and how do I test it?

Reply:
Have a look into Contract/DeptracConfig.php. There are no tests in place if I remember correctly. Right now the Php config mirrors the output of the yaml config, but with the next release the yaml config will be deprecated and the Php one will be the default.

Reply:
> This needs to be also possible via the `php-config`. Because the `yaml` config will be deprecated soon.

May I ask why? With MediaWiki, we are trying to move away from PHP config files, at least for  "simple" installations. PHP files are faster to load, which matters for latency of web API requests and such, but that isn't a concern here...

Reply:
For the same reason the symfony core prefers Php over yaml for configs.

Autocomplete and reusable parts for more complex setups.

Reply:
ack